docs/installing-environment-modules-without-root-permissions.md

```diff
@@ -1,139 +1,116 @@
 # Installing environment modules without root permissions {: #installing_env_mod_c }
 
-!!! warning
-    This page will soon replace <https://docs.easybuild.io/en/latest/Installing-environment-modules-without-root-permissions.html>.
+This short guide will explain how to install the standard environment
+modules Tcl/C software package without root permissions on a Linux or
+Mac OS X system, together with Tcl on which it depends.
 
-    **
-    It still needs to be ported from *reStructuredText* (.rst) to *MarkDown* (.md),  
-    and you can help with that!
-    **
+## Tcl
 
-    - source: [`docs/Installing-environment-modules-without-root-permissions.rst` in `easybuilders/easybuild` repo](https://raw.githubusercontent.com/easybuilders/easybuild/develop/docs/Installing-environment-modules-without-root-permissions.rst)
-    - target: [`docs/installing-environment-modules-without-root-permissions.md` in `easybuilders/easybuild-docs` repo](https://github.com/easybuilders/easybuild-docs/tree/main/docs/installing-environment-modules-without-root-permissions.md)
+1. Go to <https://www.tcl.tk> and download the latest Tcl sources. At
+    the time of writing, the latest available Tcl version was 8.5.15,
```

Comment:
should this be updated to reflect the current "as of writing" version?

Reply:
We'll handle any such updates separately.